backends-velox/src/test/scala/org/apache/gluten/execution/JsonFunctionsValidateSuite.scala

```diff
@@ -325,4 +326,64 @@ class JsonFunctionsValidateSuite extends FunctionsValidateSuite {
         }
     }
   }
+
+  test("json_object_keys function valid json format") {
+    withTempPath {
+      path =>
+        Seq[(String)](
+          (""""""),
+          ("""200"""),
+          ("""{}"""),
+          ("""{"key": 1}"""),
+          ("""{"key": "value", "key2": 2}"""),
+          ("""{"arrayKey": [1, 2, 3]}"""),
+          ("""{"key":[1,2,3,{"key":"value"},[1,2,3]]}"""),
+          ("""{"f1":"abc","f2":{"f3":"a", "f4":"b"}}"""),
+          ("""{"k1": [1, 2, {"key": 5}], "k2": {"key2": [1, 2]}}"""),
+          ("""[1, 2, 3]""")
```

Comment:
nit: these two tests can be merged into a single test to verify valid, invalid, null, and encoding cases (e.g., Chinese).

Reply:
Sure, merged.